### PR TITLE
Sanitize vhost input into sushi script

### PR DESCRIPTION
**Pull request by brettt89**
Enforce alphanumeric sanitization on input to prevent exploitation of source scripts outside intended directory.

After making change, strips the ../ characters to ensure it cannot source files outside of intended location.
```
root@smalltest2:~# vhost=../smalltest2 sushi dev/tasks
SILVERSTRIPE DEVELOPMENT TOOLS: Tasks
--------------------------

 * MigrateContentToElement: sake dev/tasks/DNADesign-Elemental-Tasks-MigrateContentToElement
 * MaximeRainville\SilverStripeGraphQLBenchmark\Tasks\SchemaCreatorTask: sake dev/tasks/MaximeRainville-SilverStripeGraphQLBenchmark-Tasks-SchemaCreatorTask
 * Migrate SiteTree Linking Task: sake dev/tasks/MigrateSiteTreeLinkingTask
 * Database Migrations: sake dev/tasks/MigrationTask
 * FTFileMakerTask: sake dev/tasks/FTFileMakerTask
 * FTPageMakerTask: sake dev/tasks/FTPageMakerTask
 * FTPageTypeCreatorTask: sake dev/tasks/FTPageTypeCreatorTask
 * Login Session Garbage Collection Task: sake dev/tasks/LoginSessionGarbageCollectionTask
 * Symbiote\QueuedJobs\Tasks\CheckJobHealthTask: sake dev/tasks/CheckJobHealthTask
 * Symbiote\QueuedJobs\Tasks\CreateQueuedJobTask: sake dev/tasks/CreateQueuedJobTask
 * Delete all queued jobs.: sake dev/tasks/Symbiote-QueuedJobs-Tasks-DeleteAllJobsTask
 * Symbiote\QueuedJobs\Tasks\ProcessJobQueueChildTask: sake dev/tasks/ProcessJobQueueChildTask
 * Symbiote\QueuedJobs\Tasks\ProcessJobQueueTask: sake dev/tasks/ProcessJobQueueTask
 * Symbiote\QueuedJobs\Tasks\PublishItemsTask: sake dev/tasks/PublishItemsTask
 * Clear InterventionBackend cache: sake dev/tasks/InterventionBackendCacheFlushTask
 * Rewrite tags to shortcodes: sake dev/tasks/TagsToShortcodeTask
 * Migrate versionedfiles: sake dev/tasks/migrate-versionedfiles
 * Deletes all temporary test databases: sake dev/tasks/CleanupTestDatabasesTask
 * Migrate File dataobjects from 3.x and successive iterations in 4.x: sake dev/tasks/MigrateFileTask
 * i18n Textcollector Task: sake dev/tasks/i18nTextCollectorTask
```

### CODE REVIEW

File: templates/sushi_script.erb [hunk 1]
```diff
@@ -11,15 +11,18 @@ if [[ -z "$vhost" ]]; then
 	exit 2
 fi
 
+# Remove non-alphanumeric characters
+VHOST_CLEAN=${vhost//[^a-zA-Z0-9]/}
```
Comment (McJoppy) [inline templates/sushi_script.erb]: Need to include `-` and `_` characters

Comment (brettt89) [reply templates/sushi_script.erb]: These characters shouldn't be needed as they are stripped by dash - https://github.com/silverstripeltd/dash/blob/master/deploynaut/code/model/DNCodeGenerator.php#L18

Comment (brettt89) [reply templates/sushi_script.erb]: Apologies just realised this was for CCL which does support those characters

**Comment by brettt89** [comment]
Add support for `-` and `_` characters for support of CCL environments

```
root@bustertest-uat:~# vhost=bustertest-uat sushi dev/tasks
SILVERSTRIPE DEVELOPMENT TOOLS: Tasks
--------------------------

 * Refresh installed package info: sake dev/tasks/UpdatePackageInfoTask
 * Populate sample data for theme demo: sake dev/tasks/CWP-CWP-Tasks-PopulateThemeSampleDataTask
 * Cleanup generated PDFs: sake dev/tasks/CleanupGeneratedPdfBuildTask
 * Migrate SiteTree Linking Task: sake dev/tasks/MigrateSiteTreeLinkingTask
 * Migrate Comment Parent classes from 3.x: sake dev/tasks/MigrateCommentParentsTask
 * SilverStripe\ContentReview\Tasks\ContentReviewEmails: sake dev/tasks/SilverStripe-ContentReview-Tasks-ContentReviewEmails
 * SilverStripe\ContentReview\Tasks\ContentReviewOwnerMigrationTask: sake dev/tasks/SilverStripe-ContentReview-Tasks-ContentReviewOwnerMigrationTask
 * Checking broken External links in the SiteTree: sake dev/tasks/CheckExternalLinksTask
 * Database Migrations: sake dev/tasks/MigrationTask
 * Remove expired share tokens: sake dev/tasks/RemoveExpiredShareTokens
 * Workflow Reminder Task: sake dev/tasks/WorkflowReminderTask
 * Symbiote\QueuedJobs\Tasks\CheckJobHealthTask: sake dev/tasks/CheckJobHealthTask
 * Symbiote\QueuedJobs\Tasks\CreateQueuedJobTask: sake dev/tasks/CreateQueuedJobTask
 * Delete all queued jobs.: sake dev/tasks/Symbiote-QueuedJobs-Tasks-DeleteAllJobsTask
 * Symbiote\QueuedJobs\Tasks\ProcessJobQueueChildTask: sake dev/tasks/ProcessJobQueueChildTask
 * Symbiote\QueuedJobs\Tasks\ProcessJobQueueTask: sake dev/tasks/ProcessJobQueueTask
 * Symbiote\QueuedJobs\Tasks\PublishItemsTask: sake dev/tasks/PublishItemsTask
 * Convert Translatable > Fluent Task: sake dev/tasks/ConvertTranslatableTask
 * Clear InterventionBackend cache: sake dev/tasks/InterventionBackendCacheFlushTask
 * Rewrite tags to shortcodes: sake dev/tasks/TagsToShortcodeTask
 * Migrate versionedfiles: sake dev/tasks/migrate-versionedfiles
 * Deletes all temporary test databases: sake dev/tasks/CleanupTestDatabasesTask
 * Migrate File dataobjects from 3.x and successive iterations in 4.x: sake dev/tasks/MigrateFileTask
 * i18n Textcollector Task: sake dev/tasks/i18nTextCollectorTask
 * UserForms EditableFormField Column Clean task: sake dev/tasks/MigrationTask
 * SilverStripe\FullTextSearch\Solr\Tasks\Solr_Configure: sake dev/tasks/Solr_Configure
 * SilverStripe\FullTextSearch\Solr\Tasks\Solr_Reindex: sake dev/tasks/Solr_Reindex
```

File: templates/sushi_script.erb [hunk 2]
```diff
@@ -11,15 +11,18 @@ if [[ -z "$vhost" ]]; then
 	exit 2
 fi
 
+# Remove non-alphanumeric characters
+VHOST_CLEAN=${vhost//[^a-zA-Z0-9-_]/}
```
Comment (McJoppy) [inline templates/sushi_script.erb]: Although the final `-` is taken literally (ie. not within a range) it looks a lot less confusing it it's escaped IMO :)

**Review by McJoppy — APPROVED** [review]

**Pull request merged**